examples/vision/randaugment.py

```diff
@@ -0,0 +1,320 @@
+command:
+"""
+
+"""shell
+pip install -U -q imgaug
+"""
+
+"""
+## Setup
+"""
+import tensorflow as tf
+
+tf.random.set_seed(42)
+
+import numpy as np
+import matplotlib.pyplot as plt
+from tensorflow.keras import layers
+
+import tensorflow_datasets as tfds
+
+tfds.disable_progress_bar()
+
+from imgaug import augmenters as iaa
+import imgaug as ia
+
+ia.seed(42)
+
+"""
+## Load the CIFAR10 dataset
+
+For this example, we will be using the [CIFAR10
+dataset](https://www.cs.toronto.edu/~kriz/cifar.html).
+"""
+
+(x_train, y_train), (x_test, y_test) = tf.keras.datasets.cifar10.load_data()
+print(f"Total training examples: {len(x_train)}")
+print(f"Total test examples: {len(x_test)}")
+
+"""
+## Define hyperparameters
+"""
+
+AUTO = tf.data.AUTOTUNE
+BATCH_SIZE = 128
+EPOCHS = 1
+
+"""
+## Initialize `RandAugment` object
+
+Now, we will initialize a `RandAugment` object from the `imgaug.augmenters` module with
+the parameters suggested by the RandAugment authors.
+"""
+
+rand_aug = iaa.RandAugment(n=3, m=7)
+
+
+def augment(images):
+    # Input to `augment()` is a TensorFlow tensor which
+    # is not supported by `imgaug`. This is why we first
+    # convert it to its `numpy` variant.
+    images = tf.cast(images, tf.uint8)
+    return rand_aug(images=images.numpy())
+
+
+"""
+## Create TensorFlow `Dataset` objects
+
+There's one problem, though. Because `RandAugment` can only process NumPy arrays, it
+cannot be applied directly as part of the `Dataset` object (which expects TensorFlow
+tensors). To make `RandAugment` part of the dataset, we need to wrap it in a [`tf.py_function`](https://www.tensorflow.org/api_docs/python/tf/py_function).
+
+A `tf.py_function` is a TensorFlow operation (which, like any other TensorFlow operation,
+takes TF tensors as arguments and returns TensorFlow tensors) that is capable of running
+arbitrary Python code. Naturally, this Python code can only be executed on CPU (whereas
+the rest of the TensorFlow graph can be accelerated on GPU), which in some  cases can
+cause significant slowdowns -- however, in this case, the `Dataset` pipeline will run
+asynchronously together with the model, and doing preprocessing on CPU will remain
+performant.
+"""
+
+train_ds_rand = (
+    tf.data.Dataset.from_tensor_slices((x_train, y_train))
+    .shuffle(BATCH_SIZE * 100)
+    .batch(BATCH_SIZE)
+    .map(lambda x, y: (tf.image.resize(x, (72, 72)), y), num_parallel_calls=AUTO)
+    .map(
+        lambda x, y: (tf.py_function(augment, [x], [tf.float32]), y),
+        num_parallel_calls=AUTO,
+    )
+    # The returned output contains an unncessary axis of
+    # 1-D and we need to remove it.
+    .map(lambda x, y: (tf.squeeze(x), y), num_parallel_calls=AUTO)
+    .prefetch(AUTO)
+)
+
+test_ds = (
+    tf.data.Dataset.from_tensor_slices((x_test, y_test))
+    .batch(BATCH_SIZE)
+    .map(lambda x, y: (tf.image.resize(x, (72, 72)), y), num_parallel_calls=AUTO)
```

Comment:
You use the value 72 in many places, consider creating an IMAGE_SIZE variable for it